lib/takeaway.rb

```diff
@@ -0,0 +1,65 @@
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_accessor :menu, :dishes_selected, :total_price, :total_price_array, :dishes_available, :dish_price
+
+  def initialize
+    @menu = Hash.new
+    @dishes_selected = []
+  end
+
+  def menu_update(dish)
+    menu["#{dish.name}"] = dish.price.to_s + "£"
+  end 
+
+  def order
+    puts self.menu
+    puts "To begin the selection, press enter. To cancel the action, type 'exit'."
+    @total_price_array = []
+
+    while gets.chomp != 'exit' 
+      puts "Please choose a dish."
+      dish = gets.chomp
+      dishes_selected << dish
+
+      puts "Please select the quantity of the dish."
+      quantity = gets.chomp
+
+      puts "Please press enter to select another dish or type 'exit'."
+
+      dish_price_calculator(dish, quantity)
+    end
+
+    puts "Here is your order."
+    puts dishes_selected
+    total_price_calculator
+  end
+
```

Comment:
OK, so I should be relying on actual methods between objects for example to make the order, then assigning the "user interaction" responsibilities to the web framework part?